playbooks/intel_ocp_provisioning.yaml

```diff
@@ -0,0 +1,90 @@
+- hosts: localhost
+  gather_facts: no
+  vars:
+    kubeconfig_path: "~/.kube/config"
+  environment:
+    KUBECONFIG: "{{ kubeconfig_path }}"
+  vars_prompt:
+    - name: "install_operators"
+      prompt: "Do you want to install operators? 'Yes' to install NFD Operator and Intel Device Plugins Operator, or 'No' to skip"
+      private: no
+    - name: "validation_feature"
+      prompt: "Which Intel feature do you want to validate? Enter 1 for Intel SGX, 2 for Intel QAT, 3 for Intel DSA, 4 for Intel GPU"
+      private: no
+  
+  tasks:
+  - name: Validate Inputs
+    block: 
+    - name: Invalid Install Operators Input
+      fail:
+        msg: "Invalid input for Install Operators. Please enter a valid option for Install Operators (Yes/No)."
+      when: install_operators not in ["Yes", "No"]
+    - name: Invalid Validation Feature Input
+      fail:
+        msg: "Invalid input for validation feature. Please enter a valid option (1-4)."
+      when: validation_feature not in ["1", "2", "3", "4"]
+
+  - name: Install Operators
+    block:
+    - name: NFD - Install NFD Operator
+      include_tasks: install_nfd_operator.yaml
+    - name: IDPO - Install Intel Device Plugins Operator
+      include_tasks: install_device_plugins_operator.yaml
+    - name: NFD - Wait until the nfd-operator-controller Deployment is available
+      k8s_info:
+        kind: Deployment
+        wait: yes
+        name: nfd-controller-manager
+        namespace: openshift-nfd
+        wait_condition:
+          type: Available
+          status: 'True'
+    - name: NFD - Create NFD discovery CR
+      k8s: 
+        state: present
+        definition: '{{ item }}'
+        wait: yes
+        wait_condition:
+          type: Available
+          status: 'False'                                                                                                                                                                                    
+      with_items: '{{ lookup("url", "https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/nfd/node-feature-discovery-openshift.yaml", split_lines=False) | from_yaml_all | list }}'
```

Comment:
Do we need a variable to choose which iOCP version we will use?

Reply:
Yes. On the dev (main) branch it will be 'main' similar to what we have for other links in readme, we will set ocp version link before cutting release with the set-version script..